Lets Do This/Networking/DSOUserManager.m

```diff
@@ -216,16 +216,12 @@ - (void)reportbackForCampaign:(DSOCampaign *)campaign fileString:(NSString *)fil
     }];
 }
 
-- (void)postAvatarImage:(UIImage *)avatarImage sendAppEvent:(BOOL)sendAppEvent completionHandler:(void(^)(NSDictionary *))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
-    [[DSOAPI sharedInstance] postAvatarForUser:[DSOUserManager sharedInstance].user avatarImage:avatarImage completionHandler:^(id responseObject) {
-        CLS_LOG(@"%@", avatarImage);
-        NSDictionary *responseDict = responseObject[@"data"];
-        self.user.avatarURL = responseDict[@"photo"];
-        NSLog(@"postAvatarImage currentUserChanged eventDispatcher");
-        [[self appDelegate].bridge.eventDispatcher sendAppEventWithName:@"currentUserChanged" body:responseDict];
-
+- (void)postAvatarImage:(UIImage *)avatarImage completionHandler:(void(^)(DSOUser *))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
+    [[DSOAPI sharedInstance] postAvatarForUser:[DSOUserManager sharedInstance].user avatarImage:avatarImage completionHandler:^(DSOUser *user) {
+        // @todo: hack for now, having trouble just updating avatar only in RN
```

Comment:
This is what the `sendAppEvent` param aimed to do, but we weren't even using within this method for the same reason . Tried creating a separate `state.photo` property in `UserView` (to avoid reloading the user's entire profile) but couldn't get it to work. This comment is more to document that I still couldn't get it to work :crying_cat_face: